docs/Working Agreement.md

```diff
@@ -0,0 +1,166 @@
+  page.
+
+## Process Lead
+
+The process lead is responsible for leading any scrum or agile practices to enable the project to move forward.
+
+- Make sure all action items are documented and ensure each has an owner and a due date and tracks the open issues.
+- Notes as needed after planning / stand-ups.
+- Make sure that items are moved to the parking lot and ensure follow-up afterwards.
+- Maintain a location showing team’s work and status and removing impediments that are blocking the team.
+- Hold the team accountable for results in a supportive fashion.
+- Make sure that project and program documentation are up-to-date.
+- Guarantee the tracking/following up on action items from retrospectives (iteration and release planning) and from
+  daily stand-up meetings.
+- Facilitate the sprint retrospective.
+- Coach Product Owner and the team in the process, as needed.
+
+## Dev Lead
+
+- Serves as team lead of the development team.
+- Works with TPM on backlog maintenance.
+- In charge of upskilling and tasking and time estimating.
+- Responsible for coaching team and engineering quality.
+- Focuses on specific technical challenges needed in project, including deeper technical explorations.
+- Responsible for architecture and leading technical discussions.
+- Works with TPM on feedback, technical gaps, and sharing.
+- Works with TPM to bring in Subject Matter Experts (SMEs) when appropriate.
+
+## Product Owner / Technical Program Manager
+
+- Responsible for turning customer and partner requirements into detailed stories with definitions of done.
+- Responsible for documenting risks appropriately, surfacing risks to the right stakeholders at the right time, and
+  managing mitigations.
+- Responsible for scoping and defining project as mapped to customer/partner business goals.
+- Actively manages the customer Product Owner and executive/business sponsor, and technical stakeholders.
+- Jointly responsible with Tech Lead for Game Plan Documentation and Review.
+- Responsible for backlog organization, grooming, and prioritization with customer/partner.
+- Facilitates customer/partner workshops, leads discussions about the problem and business goals.
+
+## Backlog management
+
+- We communicate what we are working on through the board
+- We assign ourselves a task when we are ready to work on it (not before) and move it to active
+- We capture any work we do related to the project in a user story/task
+- We close our tasks/user stories only when the deliverable has been merged to the code repository
+- We work with the TPM if we want to add a new user story to the sprint
+- If we add a new task to the board, we make sure it matches the acceptance criteria of the user story (to avoid scope
+  creep). If it doesn't match the acceptance criteria we should discuss with the TPM to see if we need a new user story.
+  for the task or if we should adjust the acceptance criteria.
+
+## Estimation
+
+- We will estimate Stories with [Fibonacci
```

Comment:
We are doing t-shirt sizing for this project, and I am generally not a proponent of story points